packages/pluggableWidgets/color-picker-web/src/components/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createElement, ReactElement, KeyboardEvent } from "react";
+
+export interface InputProps {
+    color: string;
+    disabled: boolean;
+    children: ReactElement | null;
+    onKeyUp?: (hide?: boolean) => void;
+    onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const Input = (props: InputProps): ReactElement => {
+    const { color, disabled, onKeyUp, onChange } = props;
+    const onKeyChange = (event: KeyboardEvent<HTMLInputElement>): void => {
+        if (event.key === "ArrowDown" && onKeyUp) {
```

Comment:
Are we only interested in arrow down?

Reply:
It was like this before. I don't have new requirement's, but I am ok with adding new keys if it's needed

Reply:
I don't quite know what this is for, but might be good to figure out what flow this is and confirm with @cjfhodges

Reply:
This action is like imitation of dropdown opening when you press arrow down on select input.
Same here, just an input, and when you hit arrow down it opens color picker.

I think any other keys will be unnecessary here, because it's input. @cjfhodges what do you think?

Reply:
The down key triggers the color picker overlay to load, its similar to date picker. The down key triggers the button at the end. Other keys are not necessary. The tab key allows me to navigate through the color picker